src/bin/cli.ts

```diff
@@ -112,20 +112,23 @@ const run = async function () {
     process.exit(1)
   }
 
-  const hasErrors = batches.some((batch) => batch.errors.length > 0 || batch.contentTransformErrors.length > 0)
+  const hasTransformErrors = batches.some((batch) => batch.errors.length > 0 || batch.contentTransformErrors.length > 0)
 
-  if (hasErrors) {
+  if (hasTransformErrors) {
     console.log(chalk`{bold.red The following migration has been planned but cannot be run because it contains errors}\n\n`)
   } else {
     console.log(chalk`{bold.green The following migration has been planned}\n`)
   }
   renderPlan(batches)
 
-  if (hasErrors) {
+  if (hasTransformErrors) {
     console.log(chalk`🚨  {bold.red Migration unsuccessful}`)
     process.exit(1)
   }
 
+  const serverErrorsFile = path.join(process.cwd(), `errors-${Date.now()}`)
```

Comment:
Lets put the migration name in between and end with a `.log` extension, like `errors-add-author-1511191373630.log` - WDYT?

Reply:
+1 to that, no extension files are not always naturally associated with the right editors. I'd consider `.err` as well unless we plan to put anything else than errors in there (like a detailed log of the migration's execution, with elements like `console.log` calls for example.

Reply:
@zcei what do you mean by migration name, where does that come from?

as for the file extension, I'm not sure. giving it one feels not very Unix... what do @TimBeyer and @madtrick think?

Reply:
@jfr3000 that's the migrations' file name portion `path.basename(argv.filePath, '.js')`

Reply:
I think it might make sense to give it an extension.  
What are we currently logging? Maybe we can log as http://specs.okfnlabs.org/ndjson/ and give it the ending `.ndjson`

Reply:
We're logging the complete error objects, so `.ndjson` could make sense.